documentation/WinGet-1.0.0.yaml

```diff
@@ -678,6 +699,43 @@ components:
       items:
         $ref: '#/components/schemas/Tag'
 
+    Agreement:
+      description: One Agreement entry.
+      type: object
+      properties:
+        AgreementLabel:
+          description: The label of the agreement. i.e. EULA, AgeRating, etc
+          type: string
+          minLength: 1
+          maxLength: 40
+        Agreement:
+          description: The agreement text content.
+          nullable: true
+          type: string
+          minLength: 3
+          maxLength: 10000
+        AgreementUrl:
+          description: The agreement url.
+          $ref: '#/components/schemas/Url'
+      required:
```

Comment:
Is there no "one of" concept in the schema definition?  Or should we have two subtypes and an option?